app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -4364,15 +4364,15 @@ class BrowserTabFragment :
         private fun goFullScreen() {
             omnibar.hide()
             binding.webViewFullScreenContainer.show()
-            activity?.toggleFullScreen()
+            (activity as DuckDuckGoActivity).toggleFullScreen()
```

Comment:
Is this correct? We don't necessary go to full screen only on the DuckDuckGoActivity so I take this cast can fail?. cc/ @CrisBarreiro

Reply:
Is there any activity that doesn't implement `DuckDuckGoActivity`? I thought all did. The ones that are important here -- `BrowserActivity` and `CustomTabActivity` do.

Reply:
Not all of them do. For example, `SelectedTextSearchActivity` or `FireActivity` don't. I don't think they're relevant here, but I think we should at least so a safe cast